templates/shared_services/certs/terraform/staticweb.tf

```diff
@@ -8,7 +8,7 @@ resource "azurerm_storage_account" "staticweb" {
   account_replication_type         = "LRS"
   table_encryption_key_type        = var.enable_cmk_encryption ? "Account" : "Service"
   queue_encryption_key_type        = var.enable_cmk_encryption ? "Account" : "Service"
-  enable_https_traffic_only        = true
+  https_traffic_only_enabled       = true
```

Comment:
Is this intentionally part of this change?

Reply:
Yes. I know it's not related to this PR but is a small thing that will remove an annoying warning. I can remove if needed.

Reply:
OK thats fine to leave in, just checking :-)